CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -4,17 +4,20 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 
 <hr>
 
-## v3.5.5 (TBD)
+## v3.5.5 (2022-09-15)
 
 ### Deprecations
 - Deprecated [SetKeepAlive and SetKeepAlivePeriod in limitListenerConn](https://github.com/etcd-io/etcd/pull/14366).
 
 ### Package `clientv3`
 - Fix [do not overwrite authTokenBundle on dial](https://github.com/etcd-io/etcd/pull/14132).
+- Fix [IsOptsWithPrefix returns false even if WithPrefix() is included](https://github.com/etcd-io/etcd/pull/14187).
 
 ### etcd server
+- [Build official darwin/arm64 artifacts](https://github.com/etcd-io/etcd/issues/14001).
```

Comment:
All others items are linking to PRs, but this one links to issue. If you insist on linking to issue, then you need to update both 3.6 and 3.5 changelog with the same issue link. I prefer PR links because 1. it's easy to find the related issue via PR link, and 2. different minor versions have different PR links.

What's your opinion? @spzala